test/c_interface_test.cpp

```diff
@@ -194,8 +194,10 @@ TEST(CInterface, SpecifyConsumerNullDiagnosticForValidating) {
         // TODO(antiagainst): what validation reports is not a word offset here.
         // It is inconsistent with diassembler. Should be fixed.
         EXPECT_EQ(1u, position.index);
-        EXPECT_STREQ("Nop cannot appear before the memory model instruction",
-                     message);
+        EXPECT_STREQ(
+            "Nop cannot appear before the memory model instruction\n"
+            "  OpNop ; 0x00000014\n",
```

Comment:
Let's lose the byte offset since it's a lie in the general case.

Reply:
done